src/store.js

```diff
@@ -2,6 +2,28 @@ import Vue from "vue";
 import Vuex from "vuex";
 
 Vue.use(Vuex);
+const groupRepeatedUnits = (templateArray = ['1fr']) => {
```

Comment:
I saw your comment, I do agree that splitting this out would be a little a cleaner. Thanks!